test/unit/src/site/_filters/getPaths.js

```diff
@@ -34,9 +34,9 @@ describe('getPaths', function () {
 
   it('returns all path names the post belongs to if more than 1', function () {
     const post = {
-      fileSlug: 'is-on-https',
+      fileSlug: 'lcp',
     };
-    const expected = ['lighthouse-best-practices', 'lighthouse-pwa'];
```

Comment:
note the test change, but I just found a different article in multiple paths to replace it. Seems to work?